browser/BUILD

```diff
@@ -6,6 +6,22 @@ alias(
     actual = "gui",
 )
 
+dependencies = {
```

Comment:
Accidental copy+paste?

Reply:
actually no, I omitted it not realizing it's a name used on 19 `deps = dependencies.get(`

Reply:
But there aren't actually any dependencies there, so you should just be able to remove the "deps" argument entirely.

Reply:
TBH, if you want to make this a library, i would keep it outside of `//browser` entirely. I'd put it in `//util`. If it's *just* for the browser frontend, then i would just add it to the sources for the browser, not make a separate library out of it.

Reply:
> But there aren't actually any dependencies there, so you should just be able to remove the "deps" argument entirely.

TIL, thanks @Zer0-One 

> TBH, if you want to make this a library, 

I don't, but it wouldn't build at all without it.

>  I'd put it in //util. If it's just for the browser frontend

Util shouldn't exist... it only does when people are bad at naming things.

Reply:
If you don't want a library, then just change the sources to look like this: `srcs = ["tui.cpp", "branding.h"],`

Reply:
https://bazel.build/reference/be/c-cpp#cc_binary

Bazel is a motherfucker, but it starts to make sense after a while.